### PR TITLE
Fix for doordash.com

### PR DESCRIPTION
**Pull request by ycchiam**
Fixing the issue where some of the radio selection being invisible.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2240,6 +2240,13 @@ CSS
 
 ================================
 
+doordash.com
+
+INVERT
+.fYPzgJ::after
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2246]: It seems to me that's already inverted by default 
image

Comment (ycchiam) [reply src/config/dynamic-theme-fixes.config:2246]: image

The check boxes are fine, the issue only occurs on radio selection, where users are only allowed to select one option.
Example in the screenshot is after the code change.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:2246]: Do you have a example Link to a store who has this as well which item it is?

Comment (ycchiam) [reply src/config/dynamic-theme-fixes.config:2246]: Try this, almost every item on the menu had the radio button.
https://www.doordash.com/store/super-chicken-tysons-17410/en-US

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:2246]: Aha thanks!
I already was afright of this and the CSS selector doesn't seem to be working for me due to being it randomized.
An alternative Selector can be:
```CSS
input[kind="RADIO"][id^="Toggle"]::after
```

Comment (ycchiam) [reply src/config/dynamic-theme-fixes.config:2246]: Ah, thanks for the suggestion. This is much safer way of changing it.
Will update the branch.

**Comment by Gusted** [comment]
Thanks!

**Pull request merged**